specifyweb/frontend/js_src/lib/components/Atoms/Icons.tsx

```diff
@@ -143,7 +143,7 @@ export const dialogIcons: RR<
     </span>
   ),
   success: <span className="text-green-500">{icons.checkCircle}</span>,
-  info: <span className="text-blue-500">{icons.informationCircle}</span>,
+  info: <span>{icons.informationCircle}</span>,
```

Comment:
you might not need this empty span anymore

```ts
info: icons.informationCircle,
```

Reply:
fixed